src/mempool_limits.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#ifndef BITCOIN_MEMPOOL_LIMITS_H
+#define BITCOIN_MEMPOOL_LIMITS_H
+
+#include <cstddef>
+#include <policy/policy.h>
+
+/**
+ * Options struct containing limit options for a CTxMemPool. Default constructor
+ * populates the struct with sane default values which can be modified.
+ *
+ * Most of the time, this struct should be referenced as CTxMemPool::Limits.
+ */
+struct MemPoolLimits {
```

Comment:
In commit "mempool: Introduce (still-unused) MemPoolLimits" (6843403a15cab34bc4e1e7acf7a0e509a2d437df)

Might be good to document these members using documentation that is deleted in later commit "mempool: Use m_limit for UpdateTransactionsFromBlock" (7375ec6b6ada8f45b585f272ab20f1b9c1328ed6). Like for "The maximum allowed number of transactions including the entry and its ancestors" interesting to know the count is inclusive.